examples/3.0-alpha/bbba8553-8ec1-445f-82c9-a57251dd731c.json

```diff
@@ -6,21 +6,22 @@
   ],
   "id": "urn:uuid:bbba8553-8ec1-445f-82c9-a57251dd731c",
   "type": [
-    "VerifiableCredential"
+    "VerifiableCredential",
+    "BlockcertsCredential"
```

Comment:
so just a note that Digital Bazaar removed the error on this specific item. Still I think it's good that we have the 2 types, since that is indeed the case.

Reply:
One thing I'm not really sure of is whether or not we should/need to have `BlockcertsCredential` as a type. What we're going to have is a 3rd type in most cases, because it'll be a VC, a BC, and then a specific type of credential like `GraduationCredential` or `Transcript` or something. 

Maybe it's fine to have it so we know when it's a blockcerts, and we can make that a requirement for it being a blockcerts.